superset/examples/birth_names.py

```diff
@@ -30,6 +30,7 @@
 from superset.models.slice import Slice
 from superset.utils.core import get_example_database
 
+from ..exceptions import NoDataException
```

Comment:
Can we use absolute imports? Relative imports are fragile and easily break when you move files around.

Reply:
+1, i wonder if we can add a lint rule for this (?)

Reply:
Thanks!
@mistercrunch  I couldn't find anything among lint rules. I saw it's possible to use pygrep + pre-commit. I can check it in separate PR.